File: javascript/prob_006.js
#!/usr/local/bin/node

var _ = require('./underscore');
var N = 101;
console.log(Math.pow(_.reduce(_.range(N), function(memo, num) { return memo + num; }, 0), 2) - _.reduce(_.map(_.range(N), function(x) { return x*x; }), function(memo, num) { return memo + num; }, 0));

File: java/prob_006.java
import static java.lang.System.out;

class EulerProblem006 {
    public static void main(String[] args)
    {
        final int m = 100;

        int a = 0;
        int b = 0;

        for (int i = 0; i <= m; i++)
        {
            a += (int) Math.pow(i, 2);
            b += i;
        }

        b = (int) Math.pow(b, 2);
        out.println(b - a);
    }
}